src/Raven.Server/Documents/DocumentsStorage.cs

```diff
@@ -983,6 +983,7 @@ public struct DeleteOperationResult
             long? lastModifiedTicks = null,
             ChangeVectorEntry[] changeVector = null)
         {
+            // TODO: Should we call TableValueToDocument from here? Or it should be eascape the blittable building as of the data as we don't use it here?
```

Comment:
I mean we should remove the call to TableValueToDocument down the road... it is called by Get which called by GetDocumentOrTombstone, but there is no need to build the blittable data object since we just load it to see if it there and get the etag etc... so I suggest not to use TableValueToDocument but read just what we need when reading the doc from the delete.